deploy/helm_charts/dc_website/templates/deployment.yaml

```diff
@@ -137,6 +137,8 @@ spec:
               value: {{ .Values.website.evalTool.enabled | quote }}
             - name: ENABLE_DATAGEMMA
               value: {{ .Values.website.dataGemma.enabled | quote }}
+            - name: ENABLE_EXPERIMENTS
```

Comment:
curious why reverting to separate toggles for enable_experiments and enable_datagemma?